grapher/file_grapher_test.py

```diff
@@ -1,23 +1,27 @@
 import unittest
+from grapher.file_grapher import FileGrapher
 
 
 class TestFileGrapher(unittest.TestCase):
     """
+    Tests for FileGrapher.
     """
-    # TODO(MaikuMori): Add at least unit tests.
-    # def test_filename_to_module_name(self):
-    #     cases = [
-    #         ('foo.py', 'foo'),
-    #         ('foo/bar.py', 'foo.bar'),
-    #         ('foo/__init__.py', 'foo'),
-    #         ('foo/bar/__init__.py', 'foo.bar'),
-    #         ('__init__.py', ''),
-    #     ]
-    #     for case in cases:
-    #         filename, exp_module_name = case[0], case[1]
-    #         act_module_name = filename_to_module_name(filename)
-    #         self.assertEqual(
-    #             exp_module_name,
-    #             act_module_name,
-    #             msg=('{}: {} != {}'.format(filename, exp_module_name, act_module_name))
-    #         )
+    def test_module_path_to_parent_module_name(self):
+        """ Check if parent module name is correctly detected. """
+        cases = [
+            ('foo.py', ''),
+            ('foo/bar.py', 'foo'),
+            ('foo/__init__.py', ''),
+            ('foo/bar/__init__.py', 'foo'),
```

Comment:
It's supposed to give you parent module/package name. The other part I obtain using jedi.

Here a picture with example source and output of this function + output I piece together using Jedi. The format is:
`module_path_to_parent_module_name_output|jedi_output`.

![module_function](https://cloud.githubusercontent.com/assets/28470/8056180/50603dfa-0eae-11e5-9e2c-00a4a1ce80d7.png)

Function name is maybe a bit misleading. Using the foo.bar example. If there was class difined in the `__init__.py`, I would get `bar.SomeClass` from jedi and I need to get `foo` from this function to piece together correct path.

The relevant code is [here](https://github.com/sourcegraph/srclib-python/blob/add-back-unit-tests/grapher/file_grapher.py#L215), but there are different cases how I obtain name, in most simplest case it's just `jedi_def.full_name` which will return `bar.SomeClass` for the above example.